frontend/src/metabase/lib/urls/dashboards.ts

```diff
@@ -35,3 +35,7 @@ export function embedDashboard(token: string) {
   const siteUrl = MetabaseSettings.get("site-url");
   return `${siteUrl}/embed/dashboard/${token}`;
 }
+
+export function xrayQuestion(id: CardId) {
+  return `/auto/dashboard/question/${id}`;
+}
```

Comment:
A bit on the fence here: I'd expect `xrayQuestion` to live in `urls/question`, but the URL itself probably leads to `urls/dashboards`

Reply:
Fixed, moved to questions